server/models/Profile.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  userName: {
```

Comment:
The `username` prop is already a part of the User model, please remove it here in order to avoid duplication of data :)